test/platform/github/gh-got-wrapper.spec.ts

```diff
@@ -175,13 +164,60 @@ describe('platform/gh-got-wrapper', () => {
         message: 'Resource not accessible by integration (403)',
       })
     );
-    let e;
-    try {
-      await get('some-url', {}, false);
-    } catch (err) {
-      e = err;
-    }
+    const e = await getError();
     expect(e).toBeDefined();
     expect(e.message).toEqual('integration-unauthorized');
   });
+  it('should throw for unauthorized integration', async () => {
+    const gotErr = {
+      statusCode: 403,
+      body: { message: 'Upgrade to GitHub Pro' },
+    };
+    got.mockImplementationOnce(() => Promise.reject(gotErr));
```

Comment:
Can be simplified
😅 can be simplified: https://jestjs.io/docs/en/mock-function-api.html#mockfnmockrejectedvalueoncevalue
```suggestion
    got.mockRejectedValueOnce(gotErr);
```